### ISSUE TITLE
add option to reverse image switching direction

### ISSUE THREAD

**Issue by Raoul-Kima** [deckerst/aves#593]
Problem:
An inconsistency in the user interface:
Album/Folder/grid/mosaic View shows photo1 to the left of photo2. This is basically standard across the app ecosystem. But when viewing a single image one has to swipe the other way to get from photo1 to photo2. The animation that is played when switching between images in single image view matches the swipe direction and thus also shows the images ordered the opposite way compared to grid/mosaic view.

Solution:
Add an option to reverse the direction of the image switching swipe (and animation).

**Comment by deckerst** [comment]
I don't get what you mean. Please follow instructions in `About` > `Bug Report` and paste your system information here. Also, a screen recording showing what you describe would be helpful.

**Comment by Raoul-Kima** [comment]
I just checked again and I was wrong. There is no such inconsistency in the app.
The inconsistency I saw was not caused by Aves, but by how Aves and the file-manager interacted.
I was opening an image in Aves from a file management app. In the file management app the images were sorted after file name, but in Aves they were sorted (reverse-)chronologically, leading to the inverse order. I assume it is not possible to adapt the file sorting in Aves to that of the file manager when opening a file from there? If possible that would be great! (and I could open an issue for that if desired)

Closing the issue as solved. My Apologies!
